lib/segment/src/vector_storage/quantized/quantized_scorer_builder.rs

```diff
@@ -201,6 +284,77 @@ impl<'a> QuantizedScorerBuilder<'a> {
         }
     }
 
+    fn new_quantized_scorer2<TElement, TMetric>(
```

Comment:
What's the difference with `new_quantized_scorer`? (maybe requires a commend and better naming)